README.md

```diff
@@ -1 +1,71 @@
-# Hello-Microverse
\ No newline at end of file
+My first Micronaut Project
+
+> This is my first project as a Microverse student.
+
+![screenshot]()
+
+In this project I have added Linters for HTMLand CSS. I have also added gitignore file to hide two unnecessary files
+
+## Built With
+
+- HTML
+- CSS
+- Using VS Code
+
+## Live Demo
+
+[Live Demo Link](#)
+
```

Comment:
- [ ] Your live demo link is opening your project again, kindly remove it we don't need this at this time.

Reply:
done